python/get_metrics.py

```diff
@@ -42,27 +42,65 @@ def get_tumor_cell_count(instance_dir):
 
 def get_custom_cell_count(instance_dir):
```

Comment:
This should be introduced with a different function name, because `get_custom_cell_count` is used in`swift_run_eqpy_compare.swift` and `swift_run_sweep.swift`, where the final tumor cell count is expected to be returned.